### PR TITLE
Update category, manage user

### PR DESCRIPTION
**Pull request by nguyenvandatb**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  before_action :load_user, only: :destroy
+  layout "admin"
+
+  def index
+    if params[:q].present? || params[:action_met].present?
+      @users = User.search(params[:q])
+        .send(params[:action_met])
+        .paginate page: params[:page], per_page: Settings.per_page_user_admin
+    else
+      @users = User.all.paginate page: params[:page],
```
Comment (cntt11b1) [inline app/controllers/admin/users_controller.rb]: User.paginate page: params[:page],..

**Comment by cntt11b1** [comment]
OK

File: app/views/admin/categories/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <%= provide :title, t(".title") %>
```
Comment (hungnh103) [inline app/views/admin/categories/index.html.erb]: remove sign `=`

File: app/views/admin/categories/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+<%= provide :add_button_text, t(".add_category") %>
```
Comment (hungnh103) [inline app/views/admin/categories/index.html.erb]: as above mentioned

**Pull request merged**